help.txt

```diff
@@ -13,13 +13,11 @@ asdf plugin update --all                Update all plugins
 
 
 MANAGE PACKAGES
+asdf install                            Install all the package versions listed
+                                        in the .tool-versions file 
+asdf install [<name>]                   Install one tool at the version
```

Comment:
The test below indicates that we can only use this to install a single tool at at time, not multiple which `[<name>]` implies. Perhaps just `<name>`?

Reply:
See my thoughts in comment in "conversation" tab